tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/utils/NebulaUtils.scala

```diff
@@ -0,0 +1,119 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+package com.vesoft.nebula.tools.importer.utils
+
+import com.google.common.net.HostAndPort
+import com.typesafe.config.Config
+import com.vesoft.nebula.client.meta.MetaClientImpl
+import com.vesoft.nebula.meta.{ErrorCode, TagItem}
+import org.apache.log4j.Logger
+import org.apache.spark.sql.types.{DataType}
+import org.apache.spark.sql.types.DataTypes.{
+  BooleanType,
+  DoubleType,
+  FloatType,
+  IntegerType,
+  LongType,
+  StringType
+}
+import org.apache.thrift.TException
+
+import scala.collection.JavaConverters._
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+
+object NebulaUtils {
+  private[this] val LOG = Logger.getLogger(this.getClass)
+
+  def getDataSourceFieldType(dataSourceConf: Config,
+                             nebulaConfig: Config): Map[String, DataType] = {
+    val nebulaFields = dataSourceConf.getStringList("nebula.fields")
+    val sourceFields = dataSourceConf.getStringList("fields")
+    val address      = nebulaConfig.getStringList("metaAddress")
+    val space        = nebulaConfig.getString("space")
+    val label        = dataSourceConf.getString("name")
+
+    val hostPorts: ListBuffer[HostAndPort] = new ListBuffer[HostAndPort]
+    for (i <- 0 until address.size()) {
```

Comment:
if you can know the size of the List please specified it when initial

Reply:
I removed the loop and changed the code to :
val hostPorts = address.toArray.map(addr => HostAndPort.fromString(addr.toString)).toList